src/main/resources/db/migration/common/V1_255_3_1_v59_v60_updates_user_account.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE user_account ADD COLUMN user_full_name  CHARACTER VARYING;
+update darts.user_account dl
```

Comment:
not doing any harm but dont think alias is needed

Reply:
fixed